### DISCUSSION TITLE
GORM, sdtlib and hstore (or any other extended type)

### DISCUSSION THREAD

**Discussion by uded** [jackc/pgx#1507] [Q&A] [answered]
Does anyone have a way to register specific types (like hstore) with stdlib? It's using pgtype.NewMap to register, but that one omits hstore (da-a, obviously, no OID if not created). 

Created some crazy code for this, but still no luck.

**Comment by jackc** [comment, marked as answer]
You can use `stdlib.OptionAfterConnect` to register types. But when using database/sql + stdlib everything but the core types use the text format anyay, so I wouldn't expect it to be necessary.
